libs/core/asio/include/hpx/asio/map_hostnames.hpp

```diff
@@ -46,12 +46,18 @@ namespace hpx { namespace util {
             transform_ = f;
         }
 
+        void force_ipv4(bool const& f)
```

Comment:
Why is this a `const&`?

Reply:
just c&p from the function above. Change it to only be bool